### PR TITLE
Update API version on 101-application-gateway-waf

### PR DESCRIPTION
**Pull request by MCKLMT**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Update API version
* Add US Gov deploy button
* Minor fixes

### CODE REVIEW

File: 101-application-gateway-waf/azuredeploy.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
```
Comment (bmoore-msft) [inline 101-application-gateway-waf/azuredeploy.json:2]: revert this change

Comment (MCKLMT) [reply 101-application-gateway-waf/azuredeploy.json]: Done
